app/main.py

```diff
@@ -1,3 +1,53 @@
+from __future__ import annotations
+
+
 class Distance:
-    # Write your code here
-    pass
+    def __init__(self, km: float) -> None:
+        self.km = km
+
+    def __str__(self) -> str:
+        return f"Distance: {self.km} kilometers."
+
+    def __repr__(self) -> str:
+        return f"Distance(km={self.km})"
+
+    def __add__(self, *args) -> Distance:
```

Comment:
Use `other` parameter instead of *args. You should use *args if you don't know how many non-keyword arguments will be passed in a function
In this case it makes your code much less readable

Reply:
fixed. Now it`s ok?